fpga/scripts/write_cfgmem.tcl

```diff
@@ -25,4 +25,9 @@ if {$argc < 2 || $argc > 4} {
 lassign $argv mcsfile bitfile
 
 # https://scholar.princeton.edu/jbalkind/blog/programming-genesys-2-qspi-spi-x4-flash
-write_cfgmem -format mcs -interface SPIx4 -size 256  -loadbit "up 0x0 $bitfile" -file $mcsfile -force
+
+# GENESYS II
+# write_cfgmem -format mcs -interface SPIx4 -size 256  -loadbit "up 0x0 $bitfile" -file $mcsfile -force
+
+# VC707
+write_cfgmem -format mcs -interface bpix16 -size 128 -loadbit "up 0x0 $bitfile" -file $mcsfile -force
```

Comment:
Can we make this dependent on the board variable as well? Otherwise this breaks for the Gensys II flow.